### DISCUSSION TITLE
How do I host content at /blog/ route

### DISCUSSION THREAD

**Discussion by realchandan** [nuxt/content#2480] [Q&A] [answered]
![image](https://github.com/nuxt/content/assets/93002151/e2a4f71d-23e7-42ec-a6d5-1396383d21f7)

Hello everyone,

I want to host nuxt content at /blog/ route, I have previously added `pages/blog/[...slug].vue` but that didn't work, now I have moved the files in content folder to blog subfolder and moved [...slug].vue to pages folder. this doesn't work either.

I have not done good enough search, and I'm sure there's a solution, but having spend hours on this and trying so many things I'd rather ask the community on how do I accomplish this.

Expected result -

If I go to - `http://localhost:3000/blog/about` I'd want to see the content of `about.md` file

**Comment by oripka** [comment, marked as answer]
`pages/blog/[...slug].vue` is right, if you then query like this for `content/blog/about.md`

```
const { data: page } = await useAsyncData(`blog-${route.path}`, () => queryContent(route.path).findOne())
});
```
